### DISCUSSION TITLE
병원 목록 생성 버튼 안눌림

### DISCUSSION THREAD

**Discussion by Haanpy** [changdoc/naver-vaccine-macro#248] [Q&A]
코드 다운받아서 압축해제한 후 popup html파일 실행하여
graphql result 와 left_by_count_coords Request Payload 모두 붙여넣기 하였는데
각 해당 내용 아래 버튼이 눌리지 않네요..

혹시 방법이 잘못되었나요?
크롬으로 확장프로그램 설치 한후에 코드 다운 파일 안 popup 파일로 실행하였습니다..
차근차근 해보는데 클릭이 되질 않아 진행이 안되네요

**Comment by changdoc** [comment]
popup을 직접 여는 형태가 아니고 크롬 우측 상단에 메뉴에 설치하신 프로그램을 누르면 동작하게 나옵니다.
![image](https://user-images.githubusercontent.com/1879494/128799607-d4ce95c7-f9fd-46d0-9713-3cf48ef5e4c2.png)

확장 프로그램 설치는 https://trend21c.tistory.com/1030 여기 가이드 보고 진행하신걸까요?

**Reply by Haanpy** [reply]
앗 제 실수였네요,,! 혹시 그럼 지도에서 잔여백신 클릭 후 접종 예약 신청 페이지로 넘어가면 잠시 후 시도하세요 라는 페이지가 나오는데 1시간 후에 재시도 하면 되는건가요??

**Reply by changdoc** [reply]
네 예약 신청 페이지에 짧은시간에 새로고침을 많이 하거나 여러 병원의 예약 신청 페이지를 접근하게 되면 접속 차단이 되고 1시간 후에 정상 접속 되는것으로 파악중입니다.
